htmd/simlist.py

```diff
@@ -12,6 +12,7 @@
 from htmd.molecule.molecule import Molecule
 from joblib import Parallel, delayed
 from htmd.progress.progress import ProgressBar
+import mdtraj as md
 import logging
```

Comment:
All mdtraj trajectory formats are supported by the Molecule class :/ You didn't need to do all this work. Can you pass me a .nc file? Then I can test my new simlist which should support all formats.